### PR TITLE
Typescript SDK changes and bug fixes needed for Connect

### PR DESCRIPTION
**Pull request by kev1n-peters**
- Added a WormholeTransferStandardRelayer payload type needed to parse the VAA for an automatic transfer between EVM chains
- EVM: getIsExecuted should check if the transfer is inbound queued to determine if the transfer is completed
- Solana: getIsExecuted should check if the inboxItem is released
- Solana: getIsApproved should not swallow exception
- Solana: getInboundQueuedTransfer should return null if the transfer is not queued
- Bumped SDK dependencies needed for Connect

### CODE REVIEW

**Comment by kev1n-peters** [comment]
Should probably release SDK `0.8.0` and change the dependencies in here to `^0.8`

File: evm/ts/src/ntt.ts
Comment (barnjamin) [inline evm/ts/src/ntt.ts:157]: Should we have a separate `isQueued` fn on the interface?

Comment (kev1n-peters) [reply evm/ts/src/ntt.ts:157]: Maybe it would make the code cleaner, I can add that

File: sdk/route/src/types.ts
Comment (barnjamin) [inline sdk/route/src/types.ts:178]:
```suggestion
```

File: solana/ts/sdk/ntt.ts
Comment (barnjamin) [inline solana/ts/sdk/ntt.ts:550]: always?

**Review by barnjamin — DISMISSED** [review]

File: solana/ts/sdk/ntt.ts
Comment (kev1n-peters) [reply solana/ts/sdk/ntt.ts:550]: This is hard-coded in the contract and there's no method we can call like we do to get the version

**Review by barnjamin — DISMISSED** [review]

**Review by barnjamin — APPROVED** [review]

File: sdk/route/src/manual.ts
Comment (artursapek) [inline sdk/route/src/manual.ts:165]: extremely minor comment, this could be a shared const. but im fine with it the way it is

**Review by artursapek — APPROVED** [review]

File: sdk/route/src/manual.ts
Comment (kev1n-peters) [reply sdk/route/src/manual.ts:165]: agreed, will revisit when we add more warnings. don't want to make changes now to dismiss approvals

**Pull request merged**
